### PR TITLE
okcomputer check gets a better comment

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made?

To clarify the number of workers check in the comments

## Was the documentation (README, DevOpsDocs, wiki, consul, etc.) updated?

na

## Does this change affect how this application integrates with other services?

na

### CODE REVIEW

File: config/initializers/okcomputer.rb [hunk 1]
```diff
@@ -47,6 +47,9 @@ def check
     expected_count = Settings.expected_worker_count
     actual_count = Resque.workers.count
     message = "#{actual_count} out of #{expected_count} expected workers are up."
+    # it is normal for > to happen in 2 cases:
```
Comment (jcoyne) [inline config/initializers/okcomputer.rb]: This is a little hard to read, can you move these comments into the block and say "this branch happens when..."

Comment (ndushay) [reply config/initializers/okcomputer.rb]: do you mean just before the elsif?

Comment (jcoyne) [reply config/initializers/okcomputer.rb]: yes, after the `if`

Comment (ndushay) [reply config/initializers/okcomputer.rb]: fixed per your suggestion.

**Pull request merged**